patient_case_summary/patient_case_summary.py

```
@@ -0,0 +1,788 @@
+                    ConditionInfo(
+                        code=condition_code,
+                        display=condition_display,
+                        clinical_status=clinical_status
+                    )
+                )
+
+    # Parse encounters
+    def get_encounter_date(enc):
+        period = enc.get("period", {})
+        start = period.get("start")
+        return datetime.fromisoformat(start) if start else datetime.min
```

Ruff (0.8.2)

Undefined name `datetime`

(F821)